### ISSUE TITLE
[Screen Reader-Home Page-Blog] Name Property and Control type is not defined for the 'Search Icon'  showing near Search edit box.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2626]
**User Experience:**
Users with Visual disabilities might not able to understand the controls if Name Property is not defined. Like in this scenario Name Property is not defined for the Print button. 

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to url:- https://makecode.com/
2-Navigate to the Blog control lying in the header section on the page and select it.
3-Navigate to the Search icon showing near Search editable box  on blog page.
4-Verify that Name Property and Control type is defined for  'Search Icon' or not.

**Actual Result**
Name Property and Control type is not defined for the 'Search Icon'  showing near Search edit box  as Screen Reader reads it as Text on scan mode and blank in Items mode. 

**Expected Result**
Name Property and Control type should be defined for the 'Search Icon' showing near Search edit box.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%204.1.2%20%E2%80%93%20Name,%20Role,%20Value.docx?d=w248054a65e6847719e1e242fb5025730&csf=1

**Suggested fix:**
1- Using aria-label to give an invisible name to element, where name/title property is not used 
e.g.  
'Close' button can have only ‘X’, to make it more meaningful aria-label attribute is used with ‘Close’ text to help AT’s without showing the content on UI 
 
<div id="box"> 
   This is a pop-up box. 
     <button aria-label="Close" onclick="document.getElementById('box').style.display='none';" class="closebutton"> X </button>                                          
</div> 
 
2-   Using ARIA role to expose the control type of user interface component 
e.g. 
<div role="toolbar" tabindex="0" id="customToolbar" >  
<img src="img/btn1.gif" role="button" tabindex="-1" alt="Home" id="b1" title="Home"> 
 
3-   Using ARIA attributes to expose the element’s state  
e.g. 
<div role="button" id="customToolbar" aria-checked=”” aria-disabled=”” aria-haspopup=”” aria-expanded=”” aria-selected=””> 

**Refer Attachment:**
![mas4 12_home page_blog_search icon](https://user-images.githubusercontent.com/30434417/28871201-30ab88b2-77a1-11e7-9d59-9f3a97f41ff2.jpg)

[MAS4.1.2_Home Page_Blog_Search Icon.zip](https://github.com/Microsoft/pxt/files/1193616/MAS4.1.2_Home.Page_Blog_Search.Icon.zip)

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ and bug still exists as still Screen Reader reads it as Text on scan mode and blank in Items mode.Hence Reactivated the bug and added the Active tag.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is fixed now. Hence closing the bug
